src/vcpkg/binarycaching.cpp

```diff
@@ -2154,22 +2146,26 @@ ExpectedS<DownloadManagerConfig> vcpkg::parse_download_configuration(const Optio
     parser.parse();
     if (auto err = parser.get_error())
     {
-        return Strings::concat(err->to_string(), "\nFor more information, see ", docs::assetcaching_url, "\n");
+        return msg::format(msg::msgErrorMessage)
```

Comment:
Unfortunately parser errors don't obey the convention IDEs and such respect, because they don't start with the word error. This makes output now like: 

```
PS C:\Dev\vcpkg-tool\out\build\x64-Debug-NoArtifacts> .\vcpkg.exe install zlib
error: $X_VCPKG_ASSET_SOURCES:1:13: error: unexpected arguments: 'clear' does not accept arguments
    on expression: clear;clear,bad,bad,bad
                               ^
note: See https://github.com/Microsoft/vcpkg/tree/master/docs/users/assetcaching.md for more information.
```

so this error prefix is correct but some of the other ones I put "inside" need to be removed.

Reply:
Sorry wait I think I have that backwards. Will remove the one here...